crates/biome_cli/src/cli_options.rs

```diff
@@ -46,10 +46,14 @@ pub struct CliOptions {
     #[bpaf(long("error-on-warnings"), switch)]
     pub error_on_warnings: bool,
 
-    /// Reports information using the JSON format
-    #[bpaf(long("json"), switch, hide_usage, hide)]
+    /// It prints to terminal diagnostics and summary in JSON format
```

Comment:
I find the old documentation line to be more understandable. "Report diagnostics and summary in JSON format" would also be fine.

I see three problems with the new documentation line:
- "It prints..." Who is "it"? The command-line argument isn't an actor, it's a directive. So it can't be the subject in this sentence. The other documentation lines avoid this confusion by leaving the subject out of the description. Simply saying "Print" or "Report" is more clear here. (This is slightly different from documenting functions or methods, because they can be seen as actors, though even there it's often fine to omit the subject for brevity.)
- "prints to terminal" Why is the terminal relevant? If the argument is not supplied, it also prints to terminal, right? So mentioning it here distracts from the message you're trying to convey.
- "terminal diagnostics" Terminal diagnostics are diagnostics about the terminal, or diagnostics printed in the terminal. As such, they can be read together as a single object. But of course, that's not how they're intended in this sentence. The sentence intends to say "prints to terminal, diagnostics and summary". Adding the comma would take the flow out of the sentence however, but omitting it makes the sentence hard to parse for humans, since a human reader effectively has to back-track when they realize that interpreting "terminal diagnostics" as a single object will lead to a grammatical mistake near the end of the sentence. Fortunately this is also fixed by removing the mention of the terminal, but if it had been vital to mention the terminal, this construct would've been better split into two sentences.

Not mentioning these to pick on you, of course. Hopefully they can help us all to write clearer documentation lines!

Reply:
It's OK to provide suggestions :) I am not good at writing and I am **well** aware of that 🤣